ProcessMaker/Http/Controllers/Api/EncryptedDataController.php

```diff
@@ -14,18 +14,19 @@ public function encryptText(Request $request)
     {
         // Validate request inputs
         $request->validate([
+            'uuid' => 'nullable|uuid',
```

Comment:
`uuid` must be required

Reply:
in this case no, the first time will be null